styles.css

```diff
@@ -0,0 +1,541 @@
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.flex-center {
+  text-align: center;
+  display: flex;
+  align-items: center;
+}
+
+.flex-new {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.grid {
+  display: grid;
+  align-items: center;
+  justify-content: center;
+}
+
+.widh-marg {
+  max-width: 1020px;
+  margin: auto;
+}
+
+.mb {
+  margin-bottom: 3rem;
+}
+
+.mt {
+  margin-top: 7rem;
+}
+
+* {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+li {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.hero {
+  height: 100vh;
+  background: url(assest/img/bgimg/Header-llustration-desktop.png);
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  position: relative;
+}
+
+.header-content {
+  width: 95%;
+  margin: auto;
+  padding-top: 1.5%;
+  position: fixed;
+}
+
+.logo {
+  visibility: hidden;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.nav-list {
+  display: flex;
+  flex-direction: row;
+  width: 57%;
+  gap: 35px;
+}
+
+.nav-list a {
+  color: var(--Primary);
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 20px;
+}
+
+.nav-list li a:hover {
+  color: var(--Secondary);
+}
+
+.contant {
+  flex-direction: column;
+  gap: 24px;
+  justify-content: center;
+  height: 100vh;
+}
+
+h1 {
+  font-weight: 1300;
+  font-size: 70px;
+  line-height: 70px;
+  text-align: center;
+  letter-spacing: 0.37px;
+  color: var(--Primary);
+}
+
+h2 {
+  font-weight: 1300;
+  font-size: 70px;
+  line-height: 70px;
+  text-align: center;
+  letter-spacing: 0.37px;
+  color: var(--Secondary);
+}
+
+p {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: var(--Primary);
+  flex: none;
+  flex-grow: 0;
+  max-width: 750px;
+}
+
+.social {
+  gap: 20px;
+  margin-top: 3%;
+}
+
+.next {
+  margin-top: 5%;
+  display: flex;
+}
+
+.head {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: var(--Primary);
+}
+
+.img-multi {
+  background-image: url(assest/img/recent/holder.png);
+  background-size: cover;
+  height: 260px;
+  width: 100%;
+  margin-bottom: 20px;
+}
+
+.cards {
+  display: grid;
+  align-items: center;
+  gap: 1.5rem;
+  padding: 1.5rem 0;
+  margin: 0 1rem;
+  grid-template-columns: auto auto auto;
+}
+
+.card {
+  height: 474px;
+  width: 327px;
+  flex-direction: column;
+  border: 1px solid #ebf0ee;
+  border-radius: 10px;
+}
+
+.text {
+  margin-bottom: 25px;
+  flex-direction: column;
+}
+
+h3 {
+  margin-bottom: 5%;
+}
+
+.tags ul {
+  flex-direction: row;
+  list-style: none;
+  column-gap: 5px;
+  padding: 5px;
+  margin-bottom: 20px;
+}
+
+.tag {
+  background: #ebf0ee;
+  padding: 5px;
+  border: 1px solid #fff;
+  margin-bottom: 5px;
+  color: #000;
+  border-radius: 5px;
+  font-size: 12px;
+  line-height: 16px;
+  font-weight: 600;
+}
```

Comment:
- [ ] For the badges, the `color` is `#3a4a42`. It would be nice to make it the same `color` so that your design looks identical to that of the Figma template you chose. 👍